packages/assets-controllers/src/TokenRatesController.test.ts

```diff
@@ -93,11 +96,34 @@ describe('TokenRatesController', () => {
     );
   });
 
+  it('should not poll by default', async () => {
+    jest.useFakeTimers();
```

Comment:
I ended up having more trouble with these Jest timers in later PRs. I switched them out for sinon fake timers instead, which have the `tickAsync` function that was crucial for these tests. I'll switch them out here now.

Reply:
Done in https://github.com/MetaMask/core/pull/1501/commits/9dc68179bd9cd081da58a0a3b4f2f1e7241e5ed9